scripts/pull_request_blaster_outer/001_bump_travis_ruby23_to_236.rb

```diff
@@ -0,0 +1,24 @@
+#! /usr/bin/env ruby
```

Comment:
I'm not sure I like keeping this script in here, since it's really a one-off.

Reply:
We don't have to commit them but I thought we'd mostly like need to blast the same thing in 3 months and forget what we did last time.

Reply:
Couldn't this just be rewritten to be more generic eg:

```
current_version = # load from a config…
next_version     = # load from a config…
yml["rvm"].reject! {|r| r.start_with?(current_version)}
yml["rvm"].unshift(next_version)
```

Reply:
Like when I did the extractions I created YML files for each https://github.com/juliancheal/manageiq-provider-extractions

Reply:
@juliancheal yeah, we can do that.  My intention was to make it as simple as possible.  The release repo can "know" what are the "supported rubies to test on travis" for each branch.